src/collections/community-events/bswc-reveal-video-due-date.md

```diff
@@ -0,0 +1,8 @@
+---
+title: BSWC Reveal Video Due Date
+url: https://cube.community/tournaments/bswc-2024
+category: tournament
+startDateTime: 2024-05-31T19:10:00-05:00
+ignoreStartTime: true
```

Comment:
```suggestion
useStartTime: false
```

We need to update these. Generally, these need to be changed and probably explicitly set for all entries.